src/test/java/core/basesyntax/service/impl/ReportCreatorServiceImplTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.Assert.assertTrue;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ReportCreatorService;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class ReportCreatorServiceImplTest {
+    private static ReportCreatorService reportCreatorService;
+    private static String expected1;
+    private static String expected2;

```

Comment:
each of these variables are used in one test method, so you can make it local, go through you test classes and check if you can move fields local